service/src/main/java/bio/terra/catalog/rawls/RawlsService.java

```diff
@@ -99,4 +103,44 @@ public SystemStatusSystems status() {
     }
     return result;
   }
+
+  EntitiesApi entitiesApi(AuthenticatedUserRequest user) {
+    return new EntitiesApi(getApiClient(user));
+  }
+
+  public void exportWorkspaceDataset(
+      AuthenticatedUserRequest user, String workspaceIdSource, String workspaceIdDest) {
+    try {
+      // build source name
+      WorkspaceDetails workspaceDetailsSource =
+          workspacesApi(user).getWorkspaceById(workspaceIdSource, List.of()).getWorkspace();
+      WorkspaceName workspaceNameSource =
+          new WorkspaceName()
+              .namespace(workspaceDetailsSource.getNamespace())
+              .name(workspaceDetailsSource.getName());
+
+      // build destination name
+      WorkspaceDetails workspaceDetailsDest =
+          workspacesApi(user).getWorkspaceById(workspaceIdDest, List.of()).getWorkspace();
+      WorkspaceName workspaceNameDest =
+          new WorkspaceName()
+              .namespace(workspaceDetailsDest.getNamespace())
+              .name(workspaceDetailsDest.getName());
+
+      // possible bug: empty entityType and entityNames copies all entities
+      EntityCopyDefinition body =
+          new EntityCopyDefinition()
+              .sourceWorkspace(workspaceNameSource)
+              .destinationWorkspace(workspaceNameDest)
+              .entityType("")
+              .entityNames(List.of());
+      entitiesApi(user).copyEntities(body, false);
```

Comment:
This ignores `EntityCopyResponse` which may contain conflicts information, I assume generated when exporting into a workspace that already contains some entities. I don't think we should hold up this work to handle conflicts but maybe we should have a ticket to look at adding this support in the future?

Reply:
I added a ticket for this.